tgui/packages/tgui/interfaces/ForensicScanner.tsx

```diff
@@ -53,10 +53,17 @@ export const ForensicScanner = (props) => {
               </>
             }
           >
-            {log_data.reverse().map((log) => (
+            {log_data.reverse().map((log, index) => (
               <Section
-                key={log.scan_time}
+                key={index}
                 title={`${capitalizeFirst(log.scan_target)} scan at ${log.scan_time}`}
+                buttons={
+                  <Button
+                    icon="trash"
+                    color="transparent"
+                    onClick={() => act('delete', { index: index })}
+                  />
+                }
```

Comment:
pro tip to avoid index fuckery: map to JSX nodes first, THEN reverse the array :p